Codeinwp/neve
# Fix/issue 2973

### Summary
HTML structure for the menu icon and consistent aria-label with screen reader text

### Will affect visual aspect of the product
NO

### Test instructions
1. Create a simple site with Neve
2. Check an HTML validator

Closes #2973.
